ethernet_init/README.md

```diff
@@ -0,0 +1,23 @@
+# Component for Ethernet Initialization
+
+This component initializes Ethernet driver based on Espressif IoT Development Framework Configuration.
+
+## API
+
+### Steps to enable console in an example code:
+1. Add this component to your project using the command:
+    ```bash
+    idf.py add-dependency "espressif/ethernet_init^0.0.2"
+    ```
+2. In the main file of the example, add the following line:
```

Comment:
It seems better to only refer the component manager availability and point to the example code, instead of giving instructions in the readme file.